includes/admin/class-edd-notices.php

```diff
@@ -610,6 +580,48 @@ private function add_user_action_notices( $notice = '' ) {
 						'message' => __( 'The order has been updated successfully.', 'easy-digital-downloads' )
 					) );
 					break;
+				case 'order_added' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-added',
+						'message' => __( 'Order successfully created.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'order_trashed' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-trashed',
+						'message' => __( 'The order has been moved to the trash.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'order_restored' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-restored',
+						'message' => __( 'The order has been restored.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'payment_deleted' :
+					$this->add_notice( array(
+						'id'      => 'edd-payment-deleted',
+						'message' => __( 'The payment has been deleted.', 'easy-digital-downloads' )
```

Comment:
```suggestion
						'message' => __( 'The order has been deleted.', 'easy-digital-downloads' )
```
Just suggesting that we update this, and possibly the "payment note" message below, while these lines are being updated, so that the user facing language we are using is consistent.